src/RadixDlt.NetworkGateway.GatewayApi/Exceptions/ClientClosedConnectionException.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace RadixDlt.NetworkGateway.GatewayApi.Exceptions;
+
+public sealed class ClientClosedConnectionException : KnownGatewayErrorException
+{
+    public ClientClosedConnectionException(string userFacingMessage, string internalMessage)
+        : base(499, null, userFacingMessage, internalMessage)
```

Comment:
BTW define `private const int ClientConnectionClosedHttpStatusCode = 499` to "document" this value. Actually we might want to add some short comment / docs on why we've decided to go with non-standard 499.